doc/languages-frameworks/python.section.md

```diff
@@ -174,10 +174,20 @@ following are specific to `buildPythonPackage`:
   from `build-system.requires` to `build-system`. Note that the pyproject
   format falls back to using `setuptools`, so you can use `pyproject = true`
   even if the package only has a `setup.py`. When set to `false`, you can
-  use the existing [hooks](#setup-hooks) or provide your own logic to build the
+  use the existing [hooks](#setup-hooks) or provide your own logic to build and install the
   package. This can be useful for packages that don't support the pyproject
   format. When unset, the legacy `setuptools` hooks are used for backwards
-  compatibility.
+  compatibility, however, this behavior can be modified with the `format` parameter.
+* `format`: If `pyproject` is set to true, this value will be set to `"pyproject"`. If
+  `pyproject` is set to false, `format` will be set to `"other"`. If `pyproject`
+  is unset, `format` allows for these following values:
```

Comment:
There's a bit of overlap between the descriptions of `pyproject` and `format`. Wasn't sure the best way to remedy the duplication.

Reply:
Is `format` deprecated?

Reply:
only in the sense that pyproject is the where the community is heading. There's still a long tail of packages which haven't adopted PEP517